packages/app/src/cli/services/dev.ts

```diff
@@ -55,6 +60,7 @@ async function dev(options: DevOptions) {
     tunnelPlugin,
   } = await ensureDevEnvironment(options, token)
   const apiKey = identifiers.app
+  const [adminSession, storefrontToken] = await ensureThemeExtensionTokens(options, storeFqdn)
```

Comment:
Not sure about this being optional here 🤔 
If it's going to be after `ensureDevEnvironment` then is better for this to be on line 118 where we are sure we need the tokens, instead of them being optional.

The other option is to call this on line 54, but i guess that's not possible because we need the store